docs/community/calendar.md

```diff
@@ -0,0 +1,5 @@
+# Community Calendar
+
+The Celestia Community Calendar is available for finding all the different community call events happening in Celestia's community.
+
+You can add the Community Calendar to your personal calendar [here](https://calendar.google.com/calendar/u/0?cid=Y19za2JzbjIzNWszYmlzdHNoZ3RvNmw5ODYyNEBncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
```

Comment:
Is this an existing calendar or was this calendar created for this speciically?

Reply:
It was created specifically for community events. I created it like 2-3 weeks ago